content/en/docs/zero-code/java/agent/_index.md

```diff
@@ -1,10 +1,11 @@
 ---
 title: Java Agent
 linkTitle: Agent
```

Comment:
Why remove the aliases as part of this change?

```suggestion
linkTitle: Agent
aliases:
  - /docs/java/automatic_instrumentation
  - /docs/languages/java/automatic_instrumentation
```

Reply:
the same alias exists one level up where it makes more sense